stripe/http_client.py

```diff
@@ -90,9 +96,62 @@ def __init__(self, verify_ssl_certs=True, proxy=None):
         self._proxy = proxy.copy() if proxy else None
 
     def request(self, method, url, headers, post_data=None):
```

Comment:
Do you think we could keep `request` as it is now, name the new wrapper method `request_with_retries`, and change `api_requestor.request_raw` to call `_client.request_with_retries` instead of `_client.request`?

Reply:
Yup! more explicit name sounds better to me